cookbooks/phase_transition_kinetics/phase-transition-kinetics.cc

```diff
@@ -315,6 +388,14 @@ namespace aspect
       {
         prm.enter_subsection("Phase transition kinetics");
         {
+          data_directory = Utilities::expand_ASPECT_SOURCE_DIR(prm.get("Data directory"));
+          data_filename = prm.get("Data file name");
+          prm.enter_subsection("Ascii data model");
+          {
+            prm.set("Data directory", data_directory);
+            prm.set("Data file name", data_filename);
+          }
+          prm.leave_subsection();
```

Comment:
This part should not be necessary. `profile.parse_parameters` in the next line will read and use these parameters already. Can you try if you can still change the directory and file name without these lines?